tests/schema/pass/json_schema_dialect.yaml

```diff
@@ -0,0 +1,15 @@
+openapi: 3.1.0
+info:
+  summary: Testing jsonSchemaDialect
+  title: My API
+  version: 1.0.0
+  license:
+    name: Apache 2.0
+    identifier: Apache-2.0
+jsonSchemaDialect: https://spec.openapis.org/oas/3.1/dialect/WORK-IN-PROGRESS
+components:
+  schemas:
+    WithDollarSchema:
+      $id: "locked-metaschema"
+      $schema: https://spec.openapis.org/oas/3.1/dialect/WORK-IN-PROGRESS
+paths: {}
```

Comment:
@ralfhandl I don't understand what is going on here as this file already exists on `v3.2-dev`?